zschema/keys.py

```diff
@@ -21,34 +26,42 @@ def __init__(self, port):
         self.port = str(port)
 
     def to_bigquery(self):
-        return "p%s" % self.port
+        return "p{:s}".format(self.port)
 
     def to_es(self):
         return self.port
 
     def to_proto(self):
-        return "p%s" % self.port
+        return "p{:s}".format(self.port)
 
     to_string = to_es
 
     def __eq__(self, other):
-        if type(other) == int:
-            return int(self.port).__eq__(other)
-        elif type(other) in (str, unicode):
-            return self.port.__eq__(str(other))
+        if isinstance(other, int):
+            return int(self.port) == other
+        elif isinstance(other, string_types):
+            return self.port.__eq__(other)
```

Comment:
this feels inconsistent with L41

Reply:
int doesn't have `__eq__` in Python 3, but may as well make all these `==`